src/Notes/WC_Admin_Notes_Migrate_From_Shopify.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * WooCommerce Admin: Migrate from Shopify to WooCommerce.
+ *
+ * Adds a note to ask the client if they want to migrate from Shopify to WooCommerce.
+ *
+ * @package WooCommerce Admin
+ */
+
+namespace Automattic\WooCommerce\Admin\Notes;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Admin_Notes_Migrate_From_Shopify.
+ */
+class WC_Admin_Notes_Migrate_From_Shopify {
+	/**
+	 * Note traits.
+	 */
+	use NoteTraits;
+
+	/**
+	 * Name of the note for use in the database.
+	 */
+	const NOTE_NAME = 'wc-admin-migrate-from-shopify';
+
+	/**
+	 * Get the note.
+	 */
+	public static function get_note() {
+
+		// We want to show the note after two days.
+		$two_days = 2 * DAY_IN_SECONDS;
+		if ( ! self::wc_admin_active_for( $two_days ) ) {
+			return;
+		}
+
+		$onboarding_profile = get_option( 'woocommerce_onboarding_profile', array() );
+		if (
+			! isset( $onboarding_profile['setup_client'] ) ||
+			! isset( $onboarding_profile['selling_venues'] ) ||
+			! isset( $onboarding_profile['other_platform'] )
+		) {
+			return;
+		}
+
+		// Make sure the client is not setup.
+		if ( $onboarding_profile['setup_client'] ) {
+			return;
+		}
+
+		// We will show the notification when the client already is selling and is using Shopify.
+		if (
+			'other' !== $onboarding_profile['selling_venues'] ||
+			'shopify' !== $onboarding_profile['other_platform']
+		) {
+			return;
+		}
+
+		$note = new WC_Admin_Note();
+		$note->set_title( __( 'Do you want to migrate from Shopify to WooCommerce?', 'woocommerce-admin' ) );
+		$note->set_content( __( 'Changing eCommerce platforms might seem like a big hurdle to overcome, but it is easier than you might think to move your products, customers, and orders to WooCommerce. This article will help you with going through this process.', 'woocommerce-admin' ) );
+		$note->set_type( WC_Admin_Note::E_WC_ADMIN_NOTE_INFORMATIONAL );
+		$note->set_icon( 'info' );
+		$note->set_name( self::NOTE_NAME );
+		$note->set_content_data( (object) array() );
+		$note->set_source( 'woocommerce-admin' );
+		$note->add_action(
+			'choose-niche',
```

Comment:
Copy pasta. 🍝

Reply:
Thanks, this was changed in the commit bf4fd36